indico/modules/events/abstracts/controllers/management.py

```diff
@@ -90,7 +91,25 @@ def _checkParams(self, params):
 class RHManageAbstract(RHManageAbstractBase):
     """Display abstract management page"""
     def _process(self):
-        return WPManageAbstracts.render_template('management/abstract.html', self._conf, abstract=self.abstract)
+        form = AbstractJudgmentForm(abstract=self.abstract)
+        if form.process_ajax():
+            return form.ajax_response
+        if form.validate_on_submit():
+            abstract_data = form.data
+            judgement = abstract_data.pop('judgement')
+            merge_persons = abstract_data.pop('merge_persons')
+            judge_abstract(self.abstract, abstract_data, judgement=judgement, merge_persons=merge_persons)
+        return WPManageAbstracts.render_template('management/abstract.html', self._conf, abstract=self.abstract,
+                                                 form=form)
+
+
+class RHResetAbstractJudgment(RHManageAbstractBase):
+    def _process(self):
+        # TODO: make this an AJAX RH
```

Comment:
is that really useful? it's rarely used and you'd have to refresh quite a few things on the page. a reload seems much more useful here